code/game/gamemodes/setupgame.dm

```diff
@@ -152,10 +152,6 @@
 /proc/setupcult()
 	var/random_cult = pick(all_cults)
 	var/picked_cult = new random_cult() //this seems redundent as fuck
-	log_startup_progress("Summoning eldritch horrors...")
 	if(!picked_cult)//shouldn't happen BUT JUST TO BE SURE LET US KNOW
-		log_startup_progress(" Failed to select a cult datum, Shank a coder!")
-	else
-		log_startup_progress(" [picked_cult] has risen!")
-	//todo:add adminonly datum var, check for said var here...
+		warning(" Failed to select a cult datum, Shank a coder!")
 	return picked_cult
```

Comment:
Alright, so the issue is:
[This is called many times](https://github.com/ParadiseSS13/Paradise/blob/master/code/game/gamemodes/game_mode.dm#L45-L56).
You shouldn't put code that will have an outside effect inside this proc. Instead, you should assign `cult_dat` inside `pre_setup`, which is called only once, instead of each time we want to check whether a cult gamemode can be played.